org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/XMLLanguageServer.java

```diff
@@ -224,12 +226,12 @@ public void exit(int exitCode) {
 	}
 
 	@Override
-	public TextDocumentService getTextDocumentService() {
+	public XMLTextDocumentService getTextDocumentService() {

```

Comment:
Why you change to XMLTextDocumentService?

Reply:
It is really to call `XMLTextDocumentService#triggerValidationFor(...)` from the server extension code

Reply:
Could you explain why you want to do that?

Reply:
Ok understand, you want to do that to trigger validation when classpath changed. Let me time to give you a suggestion.

Reply:
See https://github.com/eclipse/lemminx/pull/905#discussion_r501803334